worker/src/task/2-submission.ts

```
@@ -1,102 +1,188 @@
+  }
 
-    if (!submission.prUrl) {
-      console.error("[SUBMISSION] Missing PR URL in submission");
-      throw new Error("Submission is missing PR URL");
-    }
+  console.log("[SUBMISSION] Orca client initialized successfully");
+  return orcaClient;
+}
 
-    console.log("[SUBMISSION] Submission data validated successfully:", submission);
+async function makeSubmission(params: SubmissionParams): Promise<string | void> {
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)